src/panel-manager.js

```diff
@@ -9,6 +9,14 @@ genotet.panelManager = {};
 genotet.panelManager.COLLAPSED_WIDTH = 32;
 /** @const {number} */
 genotet.panelManager.TRANSITION_TIME = 300;
+/** @const {number} */
```

Comment:
Why @private? They are constants and we should use @const to indicate a constant (non-overwritable) pointer (a variable or property) according to JsDoc.